src/exchange/message.py

```diff
@@ -19,8 +19,11 @@ def validate_role_and_content(instance: "Message", *_: Any) -> None:  # noqa: AN
         if instance.tool_use:
             raise ValueError("User message does not support ToolUse")
     elif instance.role == "assistant":
-        if not (instance.text or instance.tool_use):
-            raise ValueError("Assistant message must include a Text or ToolUsage")
+        # TODO: At least in llama3.1, there's no instance.text when
+        # when only sending a system message.
```

Comment:
I tried a pending version of qwen2 which doesn't need to comment this out, as it replies "sure go ahead..." when there's only a system message.

llama3.1 instruct errors here because it replies with nothing. I think that we can either tolerate no response, or instead not send a prompt with only a system message. Curious what you think @michaelneale...

captured data from `mitmproxy --mode reverse:http://localhost:11434 -p 11435`:
![image](https://github.com/user-attachments/assets/85764380-fbed-46b7-980e-26b50b006de1)
![image](https://github.com/user-attachments/assets/0d56ddb1-fd62-4a53-a36a-b7935cc2031a)

Reply:
I think ideally would make it handle the nothing case

Reply:
ok so the tricky bit is handling this is either permanently removing what I commented out, or we need to push down context about this being the special case of only a system message. prefs?

Reply:
nevermind, from a validator we have not many choices. just commented why we aren't enforcing the condition.